manual/manual/cmds/comp.etex

```diff
@@ -132,6 +132,12 @@ The \var{src}".annot" file can be used with the emacs commands given in
 "emacs/caml-types.el" to display types and other annotations
 interactively.
 
+\item["-args" \var{filename}]
+ Read additional newline separated command line arguments from \var{filename}.
+
+\item["-args0" \var{filename}]
+ Read additional NUL separated command line arguments from \var{filename}.
```

Comment:
`NUL-separated` (as above). Similarly, is there any Unix/Windows line-ending conversion to worry about here (e.g. would the input file `foo\r\n\0bar\r\n` be read as `foo\n\0bar\n\0` on Windows?)